modules/sms_user/config/install/sms_user.settings.yml

```diff
@@ -1,6 +1,15 @@
 active_hours:
   status: false
   ranges: []
-registration_enabled: false
-allow_password: false
-new_account_message: ''
+account_registration:
+  all_unknown_numbers:
+    status: false
+    reply_message: ''
+  formatted:
+    status: false
+    incoming_messages: []
+    activation_email: true
+    reply:
+      status: false
+      message: ''
+      message_failure: ''
```

Comment:
These names are not very descriptive now that I see them in config.
`all_unknown_numbers` --> `register_unknown_numbers`
`formatted` --> `register_preformatted_message`
`activation_email` --> `send_activation_email`
`reply` --> `reply_message`